output/schema/schema.json

```diff
@@ -17080,6 +17087,256 @@
         "namespace": "_global.info"
       }
     },
+    {
+      "attachedBehaviors": [
+        "CommonQueryParameters"
+      ],
+      "body": {
+        "kind": "properties",
+        "properties": [
+          {
+            "description": "Array of wildcard (*) patterns. The request returns values for field names\nmatching these patterns in the hits.fields property of the response.",
+            "name": "fields",
+            "required": false,
+            "type": {
+              "kind": "instance_of",
+              "type": {
+                "name": "Fields",
+                "namespace": "_types"
+              }
+            }
+          },
+          {
+            "description": "KNN query to execute",
```

Comment:
Small comment: we're trying to consistently style it as kNN in docs to match the community's terminology.